setup.py

```diff
@@ -170,7 +169,7 @@ def find_version(*paths):
                 'restview',
                 'Sphinx',
                 'sphinx-rtd-theme',
-                'requests-mock'],
+                'requests-mock==1.8.0'],
```

Comment:
`==` is a bit strict. can we use `>=` or `<=`? and why need to specify version...? I think I didn't see the issue so far.

Reply:
unittests are faiing with new version (1.9.1), the last one that we used was 1.8.0 and it was working, hence pinning that version.

Reply:
but sill better to have `requests-mock<=1.8.0`? if it works.

Reply:
removed this, no longer needed as 1.9.2 works